### ISSUE TITLE
[Request]: Please update rwkv-raven-{1b5, 3b, 7b}-q8f16_0 to _1

### ISSUE THREAD

**Issue by MrCsabaToth** [mlc-ai/binary-mlc-llm-libs#26]
As far as I understood the latest APK requires `_1` models? I'd like to try the RWKV, because it didn't work with the original apk. What's the difference between the `_0` and `_1` versions, are they incompatible?

**Comment by David-Sharma** [comment]
I do not believe that there is a difference between the names. As far as I know q8f16 refers to the following q8 - quantization (8 bits) and f16 is Floating Point 16-bit precision. The _1 or _0 is more of a release identifier.

**Comment by GameOverFlowChart** [comment]
Rwkv-5 world is being trained right now, so I would prefer that we wait until it's finished and that that one gets added to Android. Until now no apk has the lib to run raven as it seems like and if we are so late already it really makes sense to wait for rwkv 5.
